erpnext/setup/doctype/item_group/item_group.json

```diff
@@ -277,7 +277,7 @@
    "export": 1,
    "print": 1,
    "report": 1,
-   "role": "Customer",
+   "role": "All",
```

Comment:
This will revert https://github.com/frappe/erpnext/commit/e816ed89e6923bc5d378ccc0440e68aa3b61d414
Maybe add a role, don't remove an existing role

Reply:
It was removed by mistake.

Reply:
Yeah but that was not the correct fix. We have to provide select permission to all in the case to fix this issue.